examples/connext_dds/dynamic_data_skip_serialization/README.md

```diff
@@ -0,0 +1,30 @@
+# Example Code: Skip Data Serialization for Data Recording
```

Comment:
Since the field is called skip_deserialization I would suggestion renaming the example to dynamic_data_skip_deserialization example
and change this line to say 'Skip Data Deserialization for Data Recording' since for recording we're not skipping serialization, we're skipping deserialization

Reply:
I use "serialization" to refer to the general process to transmit data on the wire, rather than the specific direction. I'm trying to avoid saying "serialization and deserialization".

The example skips both directions, so while the most "correct" and specific title would be "Skip Data Deserialization for Data Recording and Data Serialization for Data Replay", I think "Skip Data Serialization for Data Recording" is a way to be more general and include both.

Reply:
I figured that was the reason. I'm okay with it. I was thinking if a customer was looking at the field 'skip_deserialization' and looking for an example, the name doesn't match exactly, but hopefully they would be able to make that leap...